android/titanium/src/java/org/appcelerator/titanium/proxy/ActionBarProxy.java

```diff
@@ -427,6 +445,9 @@ public boolean handleMessage(Message msg)
 			case MSG_SET_HOME_BUTTON_ENABLED:
 				handlesetHomeButtonEnabled(msg.getData().getBoolean(HOME_BUTTON_ENABLED));
 				return true;
+			case MSG_SET_CUSTOM_VIEW:
+				handleSetCustomView(((TiViewProxy) msg.obj));
+				return true;
```

Comment:
I'm not a fan of setters without getters. Ideally, this should be a `customView` property which we should then call `handleSetCustomView()` when the property changes.